### PR TITLE
[WORKFLOWS] Fix deploy framework

### PR DESCRIPTION
**Pull request by 0xdavinchee**
## Summary
- This PR fixes several ops-scripts for the new FlowNFT addition, namely the `deploy-framework.js`, `etherscan-verify.sh`, `info-print-contract-addresses.js`, `gov-upgrade-super-token-logic.js` scripts
- It also involves some cleanup in some other script files (the documentation) as well as some updates to the contract interface files

### CODE REVIEW

File: packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperToken.sol [hunk 1]
```diff
@@ -45,6 +45,11 @@ interface ISuperToken is ISuperfluidToken, TokenInfo, IERC20, IERC777 {
         string calldata s
     ) external;
 
+    /**************************************************************************
+    * Constants
+    *************************************************************************/
+    function SUPERFLUID_NFT_DEPLOYER_LIBRARY_ADDRESS() external view returns (address);
```
Comment (hellwolf) [inline packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperToken.sol]: This should not be there, this is a detail that should not be part of the interface.

Comment (hellwolf) [reply packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperToken.sol]: The one uses this is using "SuperToken" contract, so it's not a problem for the use sites.

File: packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperTokenFactory.sol [hunk 2]
```diff
@@ -24,6 +24,16 @@ interface ISuperTokenFactory {
...
 
+    /**************************************************************************
+    * Immutable Variables
+    **************************************************************************/
+
+    /**
+     * @notice Get the canonical super token logic.
+     * @dev Is the same as `getSuperTokenLogic`
+     */
+    function _SUPER_TOKEN_LOGIC() external view returns(ISuperToken superTokenLogic);
```
Comment (hellwolf) [inline packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperTokenFactory.sol]: external & _SUPER_TOKEN_LOGIC? (the underscore)

if it's the same as `getSuperTokenLogic`, why?

File: packages/ethereum-contracts/ops-scripts/gov-upgrade-super-token-logic.js [hunk 3]
```diff
@@ -1,8 +1,9 @@
 const async = require("async");
+const fs = require("fs");
 const SuperfluidSDK = require("@superfluid-finance/js-sdk");
 const {ZERO_ADDRESS} = require("./libs/common");
 
-const MAX_REQUESTS = 100;
+const MAX_REQUESTS = 20;
```
Comment (hellwolf) [inline packages/ethereum-contracts/ops-scripts/gov-upgrade-super-token-logic.js:6]: why the change?

File: packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperToken.sol (hunk 1 above)
Comment (0xdavinchee) [reply packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperToken.sol]: will remove

File: packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperTokenFactory.sol (hunk 2 above)
Comment (0xdavinchee) [reply packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperTokenFactory.sol]: fair, will also remove this one from interface

File: packages/ethereum-contracts/ops-scripts/gov-upgrade-super-token-logic.js (hunk 3 above)
Comment (d10r) [reply packages/ethereum-contracts/ops-scripts/gov-upgrade-super-token-logic.js:6]: because a high number increases the chance of failed RPC requests, downside with minimal upside

Comment (hellwolf) [reply packages/ethereum-contracts/ops-scripts/gov-upgrade-super-token-logic.js:6]: makes sense

File: packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperTokenFactory.sol [hunk 4]
```diff
@@ -33,7 +37,8 @@ interface ISuperTokenFactory {
     function initialize() external;
 
     /**
-     * @dev Get the current super token logic used by the factory
+     * @notice Get the canonical super token logic.
+     * @dev Is the same as `_SUPER_TOKEN_LOGIC`
```
Comment (hellwolf) [inline packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperTokenFactory.sol]: remove

```
* @dev Is the same as `_SUPER_TOKEN_LOGIC`
```

**Review by hellwolf — CHANGES_REQUESTED** [review]
"disabled-test:deployment:scripts-js-truffle": "yarn run-truffle test test/ops-scripts/deployment.test.js",

Should this still be disabled?

**Comment by 0xdavinchee** [comment]
> ```
>     "disabled-test:deployment:scripts-js-truffle": "yarn run-truffle test test/ops-scripts/deployment.test.js",
> ```
> 
> Should this still be disabled?

I don't think so, not sure why this is disabled since we use truffle to deploy not HH.

**Comment by hellwolf** [comment]
> > ```
> >     "disabled-test:deployment:scripts-js-truffle": "yarn run-truffle test test/ops-scripts/deployment.test.js",
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > Should this still be disabled?
> 
> I don't think so, not sure why this is disabled since we use truffle to deploy not HH.

It was disabled because it was broken. check the git blame. If it works again, let's uncomment it.

File: packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperTokenFactory.sol (hunk 4 above)
Comment (0xdavinchee) [reply packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperTokenFactory.sol]: addressed

Comment (0xdavinchee) [reply packages/ethereum-contracts/contracts/interfaces/superfluid/ISuperTokenFactory.sol]: addressed

**Review by hellwolf — APPROVED** [review]
